src/app/terminal/fish/page.tsx

```diff
@@ -1,9 +1,9 @@
-import { CodePreview } from '@/components/CodePreview'
-import shiki from 'shiki'
+import { CodePreview } from "@/components/CodePreview";
```

Comment:
Can you take a look at your editor because it changed all single quotes to double quotes and added semicolon on line ends 🤔

Reply:
MB, forgot Prettier is enabled by default on my environment. :c